### PR TITLE
Add version: BraveSoftware.BraveBrowser-Nightly version 93.1.30.53

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26335)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Test the PR on an x86 machine.

**Comment by vedantmgoyal2009** [comment]
> Test the PR on an x86 machine.

I don't have an x86 machine. I sold it recently as it wasn't working. ☹️

File: manifests/b/BraveSoftware/BraveBrowser-Nightly/93.1.30.53/BraveSoftware.BraveBrowser-Nightly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  InstallerSwitches:
+    Silent: ''
+    SilentWithProgress: ''
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://github.com/brave/brave-browser/releases/download/v1.30.53/BraveBrowserStandaloneNightlySetup32.exe
+  InstallerSha256: AF93518F77466DFBFB56D415082C3958E3608742883ACCA659EAC9C0C0146D1D
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/BraveBrowser-Nightly/93.1.30.53/BraveSoftware.BraveBrowser-Nightly.installer.yaml:34]:
```suggestion
  InstallerUrl: https://github.com/brave/brave-browser/releases/download/v1.30.53/BraveBrowserStandaloneSilentNightlySetup32.exe
  InstallerSha256: 6F614A6D53BBE9BCFE11C199A26B998D55019C1D5889F36968882E9D03892E08
```
Try this.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/26335#discussion_r702108309

**Comment by vedantmgoyal2009** [comment]
Copied the wrong URL while creating manifest, so it was giving an error, sorry.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**